### PR TITLE
Implement creation of sub-spaces in SpaceRequest controller

### PR DESCRIPTION
**Pull request by mfrancisc**
Grant required permissions to member operator SA so that space request controller can get/list namespace and update status and finalizer for space requests.

Jira: https://issues.redhat.com/browse/ASC-257

Related PRs:
- host-operator: https://github.com/codeready-toolchain/host-operator/pull/770
- e2e: https://github.com/codeready-toolchain/toolchain-e2e/pull/685

### CODE REVIEW

**Comment by mfrancisc** [comment]
> yeah, I just checked your PR and this will be needed for sure

@MatousJobanek thanks for confirming and review, I was unsure about the `spacerequests/status` and `spacerequests/finalizers` part.

File: scripts/add-cluster.sh [hunk 1]
```diff
@@ -128,6 +142,9 @@ rules:
   - get
   - list
   - watch
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["get", "list"]
```
Comment (MatousJobanek) [inline scripts/add-cluster.sh:147]: could you please update the format here?

Comment (mfrancisc) [reply scripts/add-cluster.sh]: Done ! please see [7ee45c5](https://github.com/codeready-toolchain/toolchain-cicd/pull/88/commits/7ee45c5f61a37af5e3ba1b281b824073d8c006a9)

Sorry, I did it in sandbox-sre and forgot to apply same format here! Thanks for pointing that out!

**Comment by MatousJobanek** [comment]
@mfrancisc all related ClusterRoles were updated

**Pull request merged**